### ISSUE TITLE
@angular-devkit/build-angular 0.801.1: breakpoints not working in vscode

### ISSUE THREAD

**Issue by jmesa-sistel** [angular/angular-cli#15116]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [ ] build
- [X] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

 Yes, the previous version in which this bug was not present was: @angular-devkit/build-angular 0.801.0

### Description

 
Breakpoints are not working in vscode since @angular-devkit/build-angular 0.801.1 version
If I go back to @angular-devkit/build-angular 0.801.0 breakpoints work fine

## 🔬 Minimal Reproduction

Add breakpoints in vscode and debug

## 🔥 Exception or Error
<pre><code>

</code></pre>

## 🌍 Your Environment
<pre><code>

Angular CLI: 8.1.1
Node: 12.4.0
OS: win32 x64
Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.801.1
@angular-devkit/core         8.1.1
@angular-devkit/schematics   8.1.1
@schematics/angular          8.1.1
@schematics/update           0.801.1
rxjs                         6.4.0

</code></pre>

**Anything else relevant?**

OS: win32 x64
Not tested in linux
Tested @angular-devkit/build-angular 0.801.2 also, and breakpoints are not working

**Comment by alan-agius4** [comment]
Hi, can you please provide more information about the configuration in your `.vscode/launch.json`?

Are you following https://github.com/microsoft/vscode-recipes/tree/master/Angular-CLI and specifying the correct `sourceMapPathOverrides`?

**Comment by alan-agius4** [comment]
@jmesa-sistel, found the culprit that was causing this and made breaks work only with `sourceMapPathOverrides`

**Comment by roblourens** [comment]
What version of angular-cli will the fix go into? Can you check that it still works even with those `sourceMapPathOverrides` set? I want to make sure that our recommendation works and that we don't need to update it to say "use A for version X and B for version Y"

**Comment by alan-agius4** [comment]
`sourceMapPathOverrides` overrides will still work. The fix will be available in 8.1.2.

**Comment by CreativeHouseDotOrg** [comment]
Seems like only reverting angular back to version 7.2.15 has fixed the breakpoint issue for me and I can look in to my breakpoints again.

**Comment by jmesa-sistel** [comment]
@alan-agius4 I guess you mean 8.1.3, because the problem is in 8.1.1 & 8.1.2. For me 8.1.0 is working fine.
Will you close the issue or should I close it?

**Comment by alan-agius4** [comment]
The issue will be closed automatically once the fix is merged. Yeah indeed the fix version will be `8.1.3`.

**Comment by jmesa-sistel** [comment]
@alan-agius4  
The issue is not fixed. I need sourceMapPathOverrides to get breakpoints working.
Should I create a new issue?

**Comment by jmesa-sistel** [comment]
@alan-agius4 
More info (from `.scripts` in the debug console) using angular 8.1.3:
Working with sourceMapPathOverrides 
```
› http://127.0.0.1:4200/jira/default~editor-editor-module~src-app-plugins-field-field-module-ts~src-app-plugins-process-process-m~abd0a996.js
    - /./src/app/editor/solution-explorer/solution-explorer.component.html (c:\Proyectos\BPS\src\webapp\src\app\editor\solution-explorer\solution-explorer.component.html)
    - /./src/app/editor/menuitems/ActionContextMenuItem.ts (c:\Proyectos\BPS\src\webapp\src\app\editor\menuitems\ActionContextMenuItem.ts)
    - /./src/app/editor/menuitems/ActionMenuItemBase.ts (c:\Proyectos\BPS\src\webapp\src\app\editor\menuitems\ActionMenuItemBase.ts)
    - /./src/app/editor/services/hint.service.ts (c:\Proyectos\BPS\src\webapp\src\app\editor\services\hint.service.ts)
    - /./src/app/editor/services/solution-explorer.service.ts (c:\Proyectos\BPS\src\webapp\src\app\editor\services\solution-explorer.service.ts)
    - /./src/app/editor/solution-explorer/solution-explorer-base-node.ts (c:\Proyectos\BPS\src\webapp\src\app\editor\solution-explorer\solution-explorer-base-node.ts)
    - /./src/app/editor/solution-explorer/solution-explorer.component.scss (c:\Proyectos\BPS\src\webapp\src\app\editor\solution-explorer\solution-explorer.component.scss)
    - /./src/app/editor/solution-explorer/solution-explorer.component.ts (c:\Proyectos\BPS\src\webapp\src\app\editor\solution-explorer\solution-explorer.component.ts)
    - /./src/app/plugins/field/models/custom-fields.model.ts (c:\Proyectos\BPS\src\webapp\src\app\plugins\field\models\custom-fields.model.ts)
    - /./src/app/plugins/process/models/process.model.ts (c:\Proyectos\BPS\src\webapp\src\app\plugins\process\models\process.model.ts)
```
Not working without sourceMapPathOverrides
```
› http://127.0.0.1:4200/jira/default~editor-editor-module~src-app-plugins-field-field-module-ts~src-app-plugins-process-process-m~abd0a996.js
    - /./src/app/editor/solution-explorer/solution-explorer.component.html (/./src/app/editor/solution-explorer/solution-explorer.component.html)
    - /./src/app/editor/menuitems/ActionContextMenuItem.ts (/./src/app/editor/menuitems/ActionContextMenuItem.ts)
    - /./src/app/editor/menuitems/ActionMenuItemBase.ts (/./src/app/editor/menuitems/ActionMenuItemBase.ts)
    - /./src/app/editor/services/hint.service.ts (/./src/app/editor/services/hint.service.ts)
    - /./src/app/editor/services/solution-explorer.service.ts (/./src/app/editor/services/solution-explorer.service.ts)
    - /./src/app/editor/solution-explorer/solution-explorer-base-node.ts (/./src/app/editor/solution-explorer/solution-explorer-base-node.ts)
    - /./src/app/editor/solution-explorer/solution-explorer.component.scss (/./src/app/editor/solution-explorer/solution-explorer.component.scss)
    - /./src/app/editor/solution-explorer/solution-explorer.component.ts (/./src/app/editor/solution-explorer/solution-explorer.component.ts)
    - /./src/app/plugins/field/models/custom-fields.model.ts (/./src/app/plugins/field/models/custom-fields.model.ts)
    - /./src/app/plugins/process/models/process.model.ts (/./src/app/plugins/process/models/process.model.ts)    
```

**Comment by CreativeHouseDotOrg** [comment]
> Ohh My bad, `@angular-devkit/build-angular: 0.801.3` is not out yet.

But it's still a good question. When will it be available?

**Comment by jmesa-sistel** [comment]
@alan-agius4
Is not working for me. When I execute .scripts in debug console I can see:
`- ./src/app/shared/iframe/iframe.component.html (c:\Proyectos\BPS\src\webapp\jira\src\app\shared\iframe\iframe.component.html)`

and it should be:

`- ./src/app/shared/iframe/iframe.component.html (c:\Proyectos\BPS\src\webapp\src\app\shared\iframe\iframe.component.html)`
I can see `jira` word which is my base href in angular.json (`"baseHref": "/jira/"`)
